src/ray/raylet/node_manager.cc

```diff
@@ -2814,6 +2833,8 @@ void NodeManager::ConsumeSyncMessage(
     if (UpdateResourceUsage(node_id, data)) {
       cluster_task_manager_->ScheduleAndDispatchTasks();
     }
+    data.set_should_global_gc(false);
```

Comment:
we should move this line to line 439 so people have a better idea why we need to do that. Also add a comment saying why.

Reply:
I feel not good doing this. The logic seems fit here better right, it's more related to here instead of L439.

I can add comment in L439 for this.

Reply:
Double think about this, I think I've splitted the commands from the resource view. Let me test it. It's probably not necessary.